communitiesuk/funding-service-pre-award-stores
# FSPT-55: Application Store lift and shift

### Change description
Lift & shift of `application-store` into `pre-award-stores` and basic tweaks to make this work as a single application with `application_store` specific code in its own module.

All existing `application-store` tests pass.



### How to test
- Check out `feature/FSPT-55-update-docker-runner-for-application-store-merge` on the [docker-runner repo](https://github.com/communitiesuk/funding-service-design-docker-runner/)
- In the docker-runner repo:
- Run `./scripts/reset-all-repos.sh -m` script to pull down the latest `pre-award-stores` repo
- Run `./scripts/install-venv-all-repos.sh -p` to resync the venv and make sure pre-commit hooks are up to date 
- Stop your running containers
- Rebuild the `pre-award-stores` container with `docker compose build pre-award-stores`
- Run `make up` to create the new containers and databases
- Run things locally and check everything still works correctly (you should be able to submit and application ad see it pulled through to assess)
- Hit [http://localhost:3012/application/ui](http://localhost:3012/appliication/ui) in your browser and test the endpoints
- Run pytest and all root tests and `application_store` tests should pass.
